### DISCUSSION TITLE
Support Roam-like linking

### DISCUSSION THREAD

**Discussion by anaclumos** [facebook/docusaurus#7610] [Ideas]
Is there any way to use `/docs` folder in Roam-Research-like way?

For example
- supporting linkings with `[[Title]]`.
- supporting daily journal

**Comment by slorber** [comment]
I'd like to have bi-directional linking like Roam/Obsidian for my own use-case, but so far nobody has build this sorry

Related to this feature request: https://docusaurus.io/feature-requests/p/backlinks-or-pages-that-link-here
